src/NLog.Targets.Seq/Levels.cs

```diff
@@ -0,0 +1,101 @@
+                ["tr"] = ("Trace", LogLevel.Trace),
+                ["trc"] = ("Trace", LogLevel.Trace),
+                ["trce"] = ("Trace", LogLevel.Trace),
+                ["trace"] = ("Trace", LogLevel.Trace),
+                ["v"] = ("Verbose", LogLevel.Trace),
+                ["ver"] = ("Verbose", LogLevel.Trace),
+                ["vrb"] = ("Verbose", LogLevel.Trace),
+                ["verb"] = ("Verbose", LogLevel.Trace),
+                ["verbose"] = ("Verbose", LogLevel.Trace),
+                ["d"] = ("Debug", LogLevel.Debug),
+                ["de"] = ("Debug", LogLevel.Debug),
+                ["dbg"] = ("Debug", LogLevel.Debug),
+                ["deb"] = ("Debug", LogLevel.Debug),
+                ["dbug"] = ("Debug", LogLevel.Debug),
+                ["debu"] = ("Debug", LogLevel.Debug),
+                ["debug"] = ("Debug", LogLevel.Debug),
+                ["i"] = ("Information", LogLevel.Info),
+                ["in"] = ("Information", LogLevel.Info),
+                ["inf"] = ("Information", LogLevel.Info),
+                ["info"] = ("Information", LogLevel.Info),
+                ["information"] = ("Information", LogLevel.Info),
+                ["notice"] = ("Notice", LogLevel.Info),
+                ["w"] = ("Warning", LogLevel.Warn),
+                ["wa"] = ("Warning", LogLevel.Warn),
+                ["war"] = ("Warning", LogLevel.Warn),
+                ["wrn"] = ("Warning", LogLevel.Warn),
+                ["warn"] = ("Warning", LogLevel.Warn),
+                ["warning"] = ("Warning", LogLevel.Warn),
+                ["e"] = ("Error", LogLevel.Error),
+                ["er"] = ("Error", LogLevel.Error),
+                ["err"] = ("Error", LogLevel.Error),
+                ["erro"] = ("Error", LogLevel.Error),
+                ["eror"] = ("Error", LogLevel.Error),
+                ["error"] = ("Error", LogLevel.Error),
+                ["f"] = ("Fatal", LogLevel.Fatal),
+                ["fa"] = ("Fatal", LogLevel.Fatal),
+                ["ftl"] = ("Fatal", LogLevel.Fatal),
+                ["fat"] = ("Fatal", LogLevel.Fatal),
+                ["fatl"] = ("Fatal", LogLevel.Fatal),
+                ["fatal"] = ("Fatal", LogLevel.Fatal),
+                ["c"] = ("Critical", LogLevel.Fatal),
+                ["cr"] = ("Critical", LogLevel.Fatal),
+                ["crt"] = ("Critical", LogLevel.Fatal),
+                ["cri"] = ("Critical", LogLevel.Fatal),
+                ["crit"] = ("Critical", LogLevel.Fatal),
+                ["critical"] = ("Critical", LogLevel.Fatal),
+                ["emerg"] = ("Emergency", LogLevel.Fatal),
+                ["panic"] = ("Panic", LogLevel.Fatal)
+            };
+
+        const string LevelMarker = "\"MinimumLevelAccepted\":\"";
+
+        /// <summary>Maps a level to an NLog equivalent level.</summary>
+        /// <returns>NLog level; gauranteed not null</returns>
+        public static LogLevel ToNLogLevel(this string level)
```

Comment:
The level names returned in `MinimumLevelAccepted` will always be the ones in: https://github.com/datalust/seq-api/blob/7c11711ecb93aec14884bbc1cb2fad3f5e296f8f/src/Seq.Api/Model/LogEvents/LogEventLevel.cs#L20

Reply:
I don't think that's correct. In `Levels.cs` line 82:

```lang=cs
            return LevelsByName.TryGetValue(level, out var m) ? m.Item2 : LogLevel.Info;
```

This pulls out `Item2` of the Tuple, which I have replaced with `NLog.LogLevel` entries. Did I miss the point?

Reply:
Thanks for the follow-up! In the [Serilog version of the code](https://github.com/serilog/serilog-sinks-seq/blob/dev/src/Serilog.Sinks.Seq/Sinks/Seq/SeqApi.cs#L53), the minimum level is parsed using:

```csharp
            var value = eventInputResult.Substring(startValue, endValue - startValue);
            if (!Enum.TryParse(value, out LogEventLevel minimumLevel))
                return null;
```

The `Enum.TryParse` call only recognizes those seven levels built into Serilog; the `Levels` version here is more comprehensive, and although it will do the same thing, I think we should be able to cut it down a bit.